<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogShortcut</class>
 <widget class="QDialog" name="DialogShortcut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string comment="LBL_DLG_SHORTCUT_TITLE">Enter shortcut</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/lh3/Resources/lh_icon.png</normaloff>:/lh3/Resources/lh_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pbAssign">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string comment="LBL_ASSIGN">Assign</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string comment="LBL_REMOVE">Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string comment="LBL_CANCEL">Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string comment="LBL_DLG_SHORTCUT_SHORTCUT">Shortcut is:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leShortcut">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>19</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lShortcutExists">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string comment="LBL_DLG_SHORTCUT_EXISTS">This shortcut already exists</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbAssign</sender>
   <signal>clicked()</signal>
   <receiver>DialogShortcut</receiver>
   <slot>OnAssign()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogShortcut</receiver>
   <slot>OnRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>DialogShortcut</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnAssign()</slot>
  <slot>OnRemove()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
